app/controllers/submissions_controller.rb

```diff
@@ -1,18 +1,17 @@
 class SubmissionsController < ApplicationController
   protect_from_forgery only: []
-  before_action :set_form, only: [:new, :create]
+  before_action :set_form, only: %i[new create]
+  before_action :ensure_user, only: [:feed]
```

Comment:
Moving this /feed path behind /admin/ is preferable, as all actions auth'd user actions are there.